setup/upgrade_minor.rst

```diff
@@ -76,12 +82,45 @@ to your code to get everything working. Additionally, some features you're
 using might still work, but might now be deprecated. While that's just fine,
 if you know about these deprecations, you can start to fix them over time.
 
-Every version of Symfony comes with an UPGRADE file (e.g. `UPGRADE-4.1.md`_)
+Every version of Symfony comes with an UPGRADE file (e.g. `UPGRADE-4.4.md`_)
 included in the Symfony directory that describes these changes. If you follow
 the instructions in the document and update your code accordingly, it should be
 safe to update in the future.
 
 These documents can also be found in the `Symfony Repository`_.
 
+.. _updating-flex-recipes:
+
+3) Updating Recipes
+-------------------
+
+Over time - and especially when you upgrade to a new version of a library, an
+updating version of the :ref:`recipe <recipes-description>` may be available.
+These updates are usually minor - e.g. new comments in a configuration file - but
+it's a good idea to update the core Symfony recipes.
+
+Starting in ``symfony/flex`` version 1.6, several commands are available to help
```

Comment:
I used to say starting from
Is there a difference with starting in ?
Thank you

Reply:
Ha! It's one of those tricky English things - I'm not sure if either is correct or if both are :p. But both "sound" fine to me - I'll use `from`.